src/WaveTrackLocation.cpp

```diff
@@ -44,36 +34,18 @@ WaveTrackLocations FindWaveTrackLocations(const WaveTrack &track)
    // Add all display locations to cache
    int curpos = 0;
 
-   const WaveClip *previousClip = nullptr;
    for (const auto clip: clips) {
       for (const auto &cc : clip->GetCutLines()) {
          auto cutlinePosition =
             clip->GetSequenceStartTime() + cc->GetSequenceStartTime();
          if (clip->WithinPlayRegion(cutlinePosition)) {
              // Add cut line expander point
-             locations.emplace_back(cutlinePosition,
-                WaveTrackLocation::locationCutLine);
+             locations.emplace_back(cutlinePosition);
          }
          // If cutline is skipped, we still need to count it
          // so that curpos matches num at the end
          curpos++;
       }
-
-      if (previousClip) {
-         if (fabs(previousClip->GetPlayEndTime() - clip->GetPlayStartTime())
-            < WAVETRACK_MERGE_POINT_TOLERANCE
-         ) {
-            // Add merge point
-            locations.emplace_back(previousClip->GetPlayEndTime(),
-               WaveTrackLocation::locationMergePoint,
-               track.GetClipIndex(previousClip),
-               track.GetClipIndex(clip)
-            );
-            curpos++;
-         }
-      }
-
-      previousClip = clip;
```

Comment:
There are changes to WaveTrackLocations and the drawing of clip edges.  The bold lines will not be drawn when magnification resolves samples — I understand that this is wanted, from the issue description.   But drawing will change at lesser magnifications too.  I don’t think this was specified.

Reply:
I thought there was an agreement that there should remain nothing of merge lines anymore. The track panel looks good without them at lower magnification. Maybe I don't understand your concern.